seatunnel-connectors-v2/connector-paimon/src/main/java/org/apache/seatunnel/connectors/seatunnel/paimon/config/PaimonSinkConfig.java

```diff
@@ -77,5 +79,12 @@ public PaimonSinkConfig(ReadonlyConfig readonlyConfig) {
         this.primaryKeys = stringToList(readonlyConfig.get(PRIMARY_KEYS), ",");
         this.partitionKeys = stringToList(readonlyConfig.get(PARTITION_KEYS), ",");
         this.writeProps = readonlyConfig.get(WRITE_PROPS);
+        checkConfig();
+    }
+
+    private void checkConfig() {
+        if (this.primaryKeys.isEmpty() && "-1".equals(this.writeProps.get("bucket"))) {
```

Comment:
There's something wrong with this, because if you created the table manually instead of automatically, there's no default value for writeProps.

Reply:
So this is just a warning .   Manually created tables should be PaimonSinkWriter initialized with a warning